conf/tachyon-env.sh.template

```diff
@@ -69,7 +69,9 @@ else
       fi
     fi
   fi
-  export TACHYON_RAM_FOLDER="/mnt/ramdisk"
+  if [[ -z ${TACHYON_RAM_FOLDER} ]]; then
```

Comment:
could you add quote: `"${TACHYON_RAM_FOLDER}"`
FYI: in Tachyon we are trying to follow style guide here https://google.github.io/styleguide/shell.xml for bash scripts